lib/channel.js

```diff
@@ -0,0 +1,68 @@
+/* ################################## */
+/* ##### Channel Registration  ##### */
+/* ################################## */
+const utils = require('openhim-mediator-utils');
+var fs = require("fs");
+
+var config = JSON.parse(fs.readFileSync("config/default.json"));
```

Comment:
For this we can rather do something like I described in [this comment](https://github.com/jembi/openhim-openinfoman-config-script/pull/3#discussion_r63847235). Then we don't have to read form the file system for each request.